docs/content/getting-started/code-generation.md

```diff
@@ -138,33 +146,27 @@ $ hof gen data.cue schema.cue -T template.txt -T =debug.yaml -O out/
 
 ### Watching for Changes
 
-Use `-w`/`--watch` to watch for changes and re-render output.
+To observe for changes and re-render output, use the -`w/--watch` flag.
```

Comment:
_To observe for_ sounds weird to me. Any objection to going back to observe -> watch?

Let'a add a sentence after, like "think of this like a live-reload you find in other tools"

Reply:
```suggestion
Use the -`w/--watch` flag to watch for changes and re-render output. Think of the '-w/--watch' flag as a live-reload option that monitors your code and automatically re-renders your output when changes are detected.
```

Reply:
Use the -`w/--watch` flag to watch for changes and re-render output. Think of the `-w/--watch` flag as a live-reload option that monitors your code and automatically re-renders your output when changes are detected.


other suggestions for the second sentence
- The '-w/--watch' flag functions like a live-reload tool that provides real-time updates to your output as you change your code.
- Using the '-w/--watch' flag is like having a live-reload feature that constantly updates your output whenever a change is made to your code.
- Using the '-w/--watch' flag is like having a live-reload tool that continuously monitors your code and instantly refreshes your output when updates are made.
- The '-w/--watch' flag functions like a live-reload tool that provides real-time updates to your output as you change your code.